src/exchanges/kraken-client.js

```diff
@@ -155,6 +167,33 @@ class KrakenClient extends BasicClient {
     this._debounceSend("unsub-ticker", this._tickerSubs, false, { name: "ticker" });
   }
 
+  /**
```

Comment:
I tried to keep the non-standard spread subscription as aligned w/your current standards as possible here